index.html

```diff
@@ -876,61 +876,52 @@ <h6 class="uppercase mb0">Gill Sans</h6>
             </section>
             <section class="bg-secondary pb0">
                 <div class="container">
-                    <div class="row">
-                        <div class="col-sm-12 text-center">
-                            <h2 class="mb64 mb-xs-24">Flexible, Adaptable & Timeless.</h2>
+                    <div class="row mb64 mb-xs-24">
+                        <div class="col-sm-12 col-md-10 col-md-offset-1 text-center">
+                            <h3>Students Section
```

Comment:
Rename to`Students` and close the h3 tag.

Reply:
Done